test/unit/IndividualClaims/getClaimsToDisplay.js

```diff
@@ -35,7 +35,7 @@ describe('getClaimsToDisplay', function () {
         coverOwner.address,
         0, // productId
         ASSET.ETH,
-        [[coverAmount, timestamp + 1, coverPeriod, 0, false, 0]],
+        [[coverAmount, timestamp + 1, coverPeriod, 7, 0, false, 0]],
```

Comment:
consider using a constant test variable here for the test since it's the same throughout